core/package.json

```diff
@@ -64,11 +64,11 @@
     "reflect-metadata": "0.1.13",
     "semver": "7.3.5",
     "sequelize": "6.11.0",
-    "sequelize-typescript": "2.1.1",
+    "sequelize-typescript": "2.1.2",
     "sqlite3": "5.0.2",
     "tar": "6.1.11",
     "ts-node": "10.4.0",
-    "umzug": "v3.0.0-beta.5",
+    "umzug": "v3.0.0",
```

Comment:
Why are we using the `v` prefix exclusively for umzug? 
Also, is the explicit dependency necessary? I don't see we're using it directly, so I _believe_ it should be able to be removed like in https://github.com/grouparoo/telemetry/pull/355